tools/plugin-testlib/src/main/java/io/spine/tools/gradle/testing/BuildGradle.java

```diff
@@ -30,15 +31,17 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
- * Creates {@link #FILE_NAME} file in the root of the test project, copying it from resources.
+ * Creates {@link #BUILD_GRADLE build.gradle} or {@link #BUILD_GRADLE_KTS build.gradle.kts} file in
```

Comment:
Please also say in which case either of these two is supposed to be created/copied.

Reply:
Added a paragraph on the priority order of `build.gradle` and `build.gradle.kts`.